samcli/commands/local/invoke/cli.py

```diff
@@ -34,6 +35,9 @@
 
 
 @click.command("invoke", help=HELP_TEXT, short_help="Invokes a local Lambda function once.")
+@hook_package_id_click_option(
+    force_prepare=False, invalid_coexist_options=["t", "template-file", "template", "parameter-overrides"]
```

Comment:
Q: why not parameter overrides?

Reply:
Parameters override option is used to override the CFN template parameters, and we will not use it to override the terraform variables. Each iac hook will have its own list of options that will be used by the hook, but we will work on this feature later after the beta release